src/simple_cache_client.rs

```diff
@@ -368,6 +370,18 @@ impl SimpleCacheClient {
         Ok(response)
     }
 
+    pub async fn flush_cache(
```

Comment:
It would be very nice to have a docstring on this describing its semantics and call rate; ideally a doctest too so folks on docs.rs can be sure they're reading up-to-date examples like many of the other apis.